yarn-project/aztec-rpc/package.json

```diff
@@ -23,11 +23,6 @@
   ],
   "jest": {
     "preset": "ts-jest/presets/default-esm",
-    "globals": {
-      "ts-jest": {
-        "useESM": true
```

Comment:
reason for this?

Reply:
Sneaked this in to to get rid of jest's warnings.